utility/block.go

```diff
@@ -76,9 +75,9 @@ func (u *UtilityContext) CreateAndApplyProposalBlock(proposer []byte, maxTransac
 	// return the app hash (consensus module will get the validator set directly)
 	stateHash, err := u.Context.ComputeStateHash()
 	if err != nil {
-		log.Fatalf("Updating the app hash failed: %v. TODO: Look into roll-backing the entire commit...\n", err)
+		u.logger.Fatal().Err(err).Msg("Updating the app hash failed. TODO: Look into roll-backing the entire commit...")
 	}
-	log.Println("CreateAndApplyProposalBlock - computed state hash:", stateHash)
+	u.logger.Info().Msgf("CreateAndApplyProposalBlock - computed state hash: %s", stateHash)
```

Comment:
Should we use `.Str` here?

Doesn't have to be done in this commit. You can create a follow-up ticket to re-evalute all `Msgf` commands and change the log lines if needed. It could be a good `core starter ticket` for someone else.

Reply:
I'll create the follow-up ticket !

Reply:
Please link to it here once you do.

Reply:
Issue created: https://github.com/pokt-network/pocket/issues/479